tests/Unit/Services/DocumentSummaryWorkflowTest.php

```
@@ -243,6 +243,44 @@ public function generate(string $prompt, string $input, array $options = []): st
         $this->assertTrue($releaseCalled);
     }
 
+    public function testGenerateClassifiesAiPolicyDenialSeparatelyFromProviderFailure(): void
+    {
+        $releaseCalled = false;
+
+        $db = $this->mockDatabase();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)